pager/src/main/java/com/google/accompanist/pager/PagerState.kt

```diff
@@ -147,13 +152,26 @@ class PagerState(
         get() = layoutPages[currentLayoutPageIndex]
 
     /**
-     * The current scroll position, as a float value between `0 until pageSize`
+     * The current scroll position, as a float value between `firstPageIndex until lastPageIndex`
      */
     private inline val absolutePosition: Float
         get() = currentLayoutPage + currentLayoutPageOffset
 
+    /**
+     * The current scroll position, as a float value between `0 until pageSize`
+     */
+    val scrollPosition: Float
```

Comment:
Why has this been added? This hasn't been added on purpose, to minimize unnecessary public API.

Reply:
`PagerIndicator` needed `coerceIn` calculation, so I added it.
I will change to call `coerceIn` directly inside `PagerIndicator`.